.prettierignore

```diff
@@ -1,4 +1,574 @@
-packages/nodes-base
 packages/editor-ui
 packages/design-system
 *package.json
+
+packages/nodes-base/*
```

Comment:
shouldn't this be removed then? or does it just affect "root" level files?

Reply:
You were right, this line was redundant. Removed.